TheJumpCloud/jumpcloud-ADMU
# CUT-4212 UWP Freeze Bug Fix

## Issues
* [CUT-4212](https://jumpcloud.atlassian.net/browse/CUT-4212) - CUT-4212 UWP app freeze on interaction

## What does this solve?
* Fixes issue with UWP app/form freezing on interaction such as clicks
* Updated useragent wording
## Is there anything particularly tricky?
N/a
## How should this be tested?

Note: 

- Test on different displays, ie. 1920x1080 and 2560x1440 to make sure the logo and texts are uniform

1. After a successful ADMU migration, build the uwp_jcadmu.exe
2. Copy the .exe file to the system folder
3. Login to the migrated account
4. When the form pops up, try to interact by clicking or typing on your keyboard. The app should not freeze or error `JumpCloud ADMU UWP Fix is not responding`. 

## Screenshots
Before:
![image](https://github.com/user-attachments/assets/a450faba-2f60-42a4-bf4a-b471fa9531be)

After:
![image](https://github.com/user-attachments/assets/20e0abc9-2e21-43e4-92bf-b69c48858c78)


[CUT-4212]: https://jumpcloud.atlassian.net/browse/CUT-4212?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
